src/utils/CookieUtils.ts

```diff
@@ -16,7 +16,7 @@ export class Cookie {
     host = location.hostname;
     if (host.split('.').length === 1) {
       // no '.' in a domain - it's localhost or something similar
-      document.cookie = this.prefix + name + '=' + value + expires + '; path=/';
+      document.cookie = this.prefix + name + '=' + value + expires + '; SameSite=Lax; path=/';
```

Comment:
Rather than duplicating this line, it would be better to place it in a function and reuse it.